### ISSUE TITLE
Interpolation problems in `Ysabeau-Italic[wght].ttf`

### ISSUE THREAD

**Issue by behdad** [CatharsisFonts/Ysabeau#49]
Hello!

This is an automatically-generated report about possible interpolation problems in `Ysabeau-Italic[wght].ttf`, as found in the Google Fonts catalog.

The particular version of the font that was tested was [archive:](archive:).

To download a PDF version of this report with helpful visuals of the problems, click [here](https://github.com/behdad/google-fonts-ofl-interpolatable-results/raw/8ce3e0535f17cd069e91df45e252462bf74a2cc4/reports/Ysabeau-Italic%5Bwght%5D.ttf.pdf); Or to view it on the GitHub website, click [here](https://github.com/behdad/google-fonts-ofl-interpolatable-results/blob/8ce3e0535f17cd069e91df45e252462bf74a2cc4/reports/Ysabeau-Italic%5Bwght%5D.ttf.pdf).

The report follows:
```
Glyph dollar.BRACKET.varAlt01 was not compatible:
  Masters: '', 'wght=1.0':
    Contour order differs: [0, 1, 2] in '', [0, 2, 1] in 'wght=1.0'
    Contour 1 start point differs: 0 in '', 2 in 'wght=1.0'; reversed: False
```

This report was generated using the `fonttools varLib.interpolatable` tool.  We understand that sometimes the tool generates false-positives.  Particularly for more complicated font designs.  If you did not find this report useful, please apologize and ignore & close it.

To give feedback about this report, please file an issue or open a discussion at [fonttools](https://github.com/fonttools/fonttools).

**Comment by CatharsisFonts** [comment]
Hi Behdad, 

thanks for your error hunting! I fixed them on my side, but then had trouble pushing them because I hadn't pulled beforehand. Trying to fix that, I maneuvered myself into a ditch and don't know how to get back out. What in the nine hells does «fix them up in the work tree» mean?

Cheers, Christian

(base) [Andromeda-2: GitHub] $ git push origin master
To https://github.com/CatharsisFonts/Ysabeau.git
 ! [rejected]        master -> master (fetch first)
error: failed to push some refs to 'https://github.com/CatharsisFonts/Ysabeau.git'
hint: Updates were rejected because the remote contains work that you do
hint: not have locally. This is usually caused by another repository pushing
hint: to the same ref. You may want to first integrate the remote changes
hint: (e.g., 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.
(base) [Andromeda-2: GitHub] $ git pull
remote: Enumerating objects: 28, done.
remote: Counting objects: 100% (28/28), done.
remote: Compressing objects: 100% (5/5), done.
remote: Total 16 (delta 10), reused 15 (delta 10), pack-reused 0
Unpacking objects: 100% (16/16), 157.00 KiB | 859.00 KiB/s, done.
From https://github.com/CatharsisFonts/Ysabeau
   aef098a..9386c5f  master     -> origin/master
hint: You have divergent branches and need to specify how to reconcile them.
hint: You can do so by running one of the following commands sometime before
hint: your next pull:
hint: 
hint:   git config pull.rebase false  # merge
hint:   git config pull.rebase true   # rebase
hint:   git config pull.ff only       # fast-forward only
hint: 
hint: You can replace "git config" with "git config --global" to set a default
hint: preference for all repositories. You can also pass --rebase, --no-rebase,
hint: or --ff-only on the command line to override the configured default per
hint: invocation.
fatal: Need to specify how to reconcile divergent branches.
(base) [Andromeda-2: GitHub] $ git config pull.rebase false
(base) [Andromeda-2: GitHub] $ git pull
Auto-merging sources/Ysabeau-Italic.glyphs
CONFLICT (content): Merge conflict in sources/Ysabeau-Italic.glyphs
Auto-merging sources/Ysabeau.glyphs
CONFLICT (content): Merge conflict in sources/Ysabeau.glyphs
Automatic merge failed; fix conflicts and then commit the result.
(base) [Andromeda-2: GitHub] $ git config pull.rebase true
(base) [Andromeda-2: GitHub] $ git pull
error: Pulling is not possible because you have unmerged files.
hint: Fix them up in the work tree, and then use 'git add/rm <file>'
hint: as appropriate to mark resolution and make a commit.
fatal: Exiting because of an unresolved conflict.
(base) [Andromeda-2: GitHub] $

**Comment by CatharsisFonts** [comment]
I think I fixed it. Can you check?

**Comment by behdad** [comment]
No looks like you got into a conflict. If you don't mind redoing the fixing, you can do:
```
git reset --hard
git pull
```
Then do your work again, then `git push`. Resolving .glyphs-file conflicts is very hard.

**Comment by CatharsisFonts** [comment]
Git still complained about some things not being compatible. I went to war with it until it accepted the changes. Now my Glyphs source files are corrupt. I am about to throw my laptop out of the window.

**Comment by behdad** [comment]
Blame @yanone

**Comment by CatharsisFonts** [comment]
I sent him a mail.

I can't even open older commits of Ysabeau.glyphs anymore that I'm downloading directly from the github website. I'm getting this error message. WFT? How can I possibly have corrupted old commits retroactively? Or is my local Glyphs app broken all of a sudden?
<img width="847" alt="Screenshot 2023-11-25 at 16 40 02" src="https://github.com/CatharsisFonts/Ysabeau/assets/13754106/5fc4d233-c960-402d-833d-23f15ea40551">

**Comment by behdad** [comment]
Glyphs is beyond me...

**Comment by CatharsisFonts** [comment]
Can _you_ open Ysabeau.glyphs from the current commit?

**Comment by behdad** [comment]
I'm on Linux currently. Don't have Glyphs.

**Comment by CatharsisFonts** [comment]
Could you revert the repo to yanone's last commit (9386c5f), though? I'm too dumb to do it myself.

**Comment by behdad** [comment]
I don't have write access to your repo. But try:
```
git reset --hard 9386c5f4b79c70a60dab5d217b1d5c1782613bec
```
see if that helps.

Just don't push anything until you figure it out.

**Comment by behdad** [comment]
Do you want the files *before* https://github.com/CatharsisFonts/Ysabeau/commit/9386c5f4b79c70a60dab5d217b1d5c1782613bec ?

**Comment by CatharsisFonts** [comment]
No, those _at_ 9386c5f. I'm assuming the commit was healthy before my Merge Of Doom.

**Comment by CatharsisFonts** [comment]
Whoa wait, it works, I can open my source file again! Do I not have to download the files again because my local instance remembers what the files were like at that stage...?

In any case, if I make changes now and push, it will generate another conflict with the more recent commits, right? How can I force my new version through that resistance?

**Comment by behdad** [comment]
If you managed to get to a good state, then you can, gasp, do:
```
git push --force
```

**Comment by CatharsisFonts** [comment]
Phew! OK, the newer commits are gone. Thanks so much!

**Comment by CatharsisFonts** [comment]
I redid the fixes in the source files, but I realized you were taking about the variable fonts in the /googlefonts directory, rather than the ones I made myself with Glyphs (in the /vf directory). I actually don't know how the Google variable fonts are exported, so you'll have to ask @yanone to regenerate them. (Or to teach me how to do it I guess?)

Cheers, Christian

**Comment by behdad** [comment]
I'll test your new ones as well.

I let @yanone figure out how to get them in the Google Fonts directory. Thanks!

**Comment by CatharsisFonts** [comment]
I actually deleted my homebrew variable fonts from the repo since presumably they're less refined than the Google ones...
